index.test.js

```diff
@@ -0,0 +1,98 @@
+const skeletorLocalServer = require('./index');
+
+jest.mock('express');
+
+global.console = {
+    log: jest.fn()
+};
+
+const logger = {
+    info: () => {},
+    error: () => {}
+};
+const options = {logger};
+let consoleSpy;
+
+describe('local server plugin', () => {
+
+    beforeEach(() => {
+        consoleSpy = jest.spyOn(global.console, 'log');
+    });
+
+    afterEach(() => {
+        consoleSpy.mockReset();
+    });
+
+    it('should error when no entry point specified', async () => {
+        const expectedError = {
+            status: 'error',
+            message: 'Error with config. Directory: testDir, Entry: undefined'
+        };
+        const config = {
+            currentDirectory: 'testDir'
+        };
+        await expect(skeletorLocalServer().run(config, options)).rejects.toEqual(expectedError);
+    });
+
+    it('should error when no entry point specified', async () => {
```

Comment:
I think this test is supposed to read 'should error when no current directory specified'